src/test/scala/edu/ie3/simona/model/participant/FixedFeedInModelSpec.scala

```diff
@@ -13,21 +13,34 @@ import edu.ie3.simona.test.common.UnitSpec
 import edu.ie3.simona.test.common.input.FixedFeedInputTestData
 import edu.ie3.util.quantities.PowerSystemUnits
 import edu.ie3.util.quantities.PowerSystemUnits.MEGAVOLTAMPERE
-import edu.ie3.util.scala.quantities.{Kilovoltamperes, Megavoltamperes}
+import edu.ie3.util.scala.quantities.{
+  ApparentPower,
+  Kilovoltamperes,
+  Megavoltamperes,
+}
+import squants.energy.Kilowatts
+import squants.{Dimensionless, Each, Power}
+import tech.units.indriya.quantity.Quantities
 
 class FixedFeedInModelSpec extends UnitSpec with FixedFeedInputTestData {
 
+  implicit val powerTolerance: Power = Kilowatts(1e-9)
+  implicit val apparentPowerTolerance: ApparentPower = Megavoltamperes(1e-9)
+  implicit val doubleTolerance: Double = 1e-9
```

Comment:
With scala3 this is working but implict val will phase out. So we should switch to use `protected given` here as well. [More information can be found here](https://rockthejvm.com/articles/scala-3-givens-vs-implicits)

Reply:
Same for the other classes